Appraisals

```diff
@@ -1,37 +1,55 @@
 # frozen_string_literal: true
 
-appraise "rails-6.1-sidekiq-5.2" do
+appraise "rails-6.1-sidekiq-6.4.1" do
   gem "activejob", "~> 6.1.0"
   gem "activesupport", "~> 6.1.0"
-  gem "sidekiq", "~> 5.2.0"
+  gem "sidekiq", "~> 6.4.1"
```

Comment:
If we have `~>` for this won't it also be the same version as the 6.4.2 group below? Same for the others.

Reply:
You are absolutely right! Good catch. I will fix that. ;)